### ISSUE TITLE
npm --ignore-prepublish

### ISSUE THREAD

**Issue by MattWellie** [igorshubovych/markdownlint-cli#650]
I've run into a pre-commit environment setup issue with `v0.49.1` which was solved on `v0.49.0`

```
[INFO] Installing environment for https://github.com/igorshubovych/markdownlint-cli.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/Users/matwel/.cache/pre-commit/repos0mir0ln/node_env-system/bin/node', '/opt/homebrew/bin/npm', 'install', '--include=dev', '--include=prod', '--ignore-prepublish', '--no-progress', '--no-save')
return code: 1
stdout: (none)
stderr:
    npm error code EUNKNOWNCONFIG
    npm error Unknown cli flag:
    npm error   - --ignore-prepublish
    npm error Run `npm help config` for supported options.
    npm error A complete log of this run can be found in: /Users/matwel/.npm/_logs/2026-08-10T01_53_46_805Z-debug-0.log
```

This occurred on an earlier version, upgrading to `v0.49.0` solved the issue, but it re-occurs in `0.49.1`

FYI 
```
❯ npm --version
12.0.2
```

**Comment by DavidAnson** [comment]
This appears to be an issue with the pre-commit tool incorrectly invoking npm. There is a discussion of the issue and a possible fix in their project: https://github.com/pre-commit/pre-commit/issues/3517
